src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -2162,6 +2229,7 @@ namespace SystemAvailabilityManager {
                                 if (state.dataHeatBalFanSys->TempTstatAir(ZoneNum) <
                                     state.dataHeatBalFanSys->TempZoneThermostatSetPoint(ZoneNum) - TempTol) {
                                     AvailStatus = CycleOn;
+                                    break;
```

Comment:
Github is confused here, it is comparing against the wrong code. This `break` statement is [already in the develop branch](https://github.com/NREL/EnergyPlus/blob/1ca392b3a183d24d4a2823e056dbb8b341b4087b/src/EnergyPlus/SystemAvailabilityManager.cc#L2266).